superset/op/superset_config_docker.py

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
I didn't dig much, but do we need all three superset config python files? I'm not sure if one or more of them might be trash.

Reply:
I believe one can go, it was a failed attempt.  The other two can probably get merged, I haven't figured out what the correct setup should be yet.